File: C++/1173.cpp
#include<iostream>

using namespace std;

int main(){

    int vetor[10];
    int entrada, valor;

    cin >> entrada;

    valor = entrada;


    for(int i = 0; i < size(vetor); i++){
        vetor[i] = valor;
        valor *= 2;
        cout << "N[" << i << "] = " << vetor[i] << endl;
    }

    return 0;
}

File: JavaScript/1173.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');
/*Começa aqui */
let vetor = [];
let entrada = parseInt(lines[0]);
let valor;

valor = entrada;

for(let i = 0; i < 10; i++){
    vetor[i] = valor;
    valor *= 2;
    console.log(`N[${i}] = ${vetor[i]}`)
}